eo-maven-plugin/src/test/java/org/eolang/maven/optimization/OptCachedTest.java

```diff
@@ -74,6 +74,44 @@ void returnsFromCacheIfXmlAlreadyInCache(@TempDir final Path tmp) throws IOExcep
         );
     }
 
+    @Disabled
+    @Test
+    void returnsFromCacheButTimesSaveAndExecuteDifferent(@TempDir final Path tmp)
+        throws IOException, InterruptedException {
+        final XML program = OptCachedTest.program(ZonedDateTime.now());
+        Thread.sleep(70_000);
+        OptCachedTest.save(tmp, program);
+        MatcherAssert.assertThat(
+            "We expected that the program will be returned from the cache",
+            new OptCached(
+                path -> {
+                    throw new IllegalStateException("This code shouldn't be executed");
+                },
+                tmp
+            ).apply(program),
+            Matchers.equalTo(program)
+        );
+    }
+
+    @Disabled
+    @Test
+    void returnsFromCacheCorrectProgram(@TempDir final Path tmp)
+        throws IOException, InterruptedException {
+        XML program = OptCachedTest.programWithSomethings(ZonedDateTime.now(), "first program");
+        OptCachedTest.save(tmp, program);
+        program = OptCachedTest.programWithSomethings(ZonedDateTime.now(), "second program");
+        MatcherAssert.assertThat(
+            "We expected that the correct program will be returned from the cache",
```

Comment:
@Yanich96, this message doesn't provide useful information for a developer. Perhaps you could explain what you specifically expect in this test case and add more context?

Reply:
@volodya-lombrozo is the message normal?
"We expected that the correct program will be returned from the cache, " +
 "but time verification is not reliable"

Reply:
@Yanich96 Yes, I like it.